uwblueprint/childrens-aid-society
# Create addresses model

## Notion ticket link

[https://www.notion.so/uwblueprintexecs/Create-addresses-model-11e335fdac594c7bbbebdde90723294e]



## Implementation description
* Created address,py file inside models
* Created the model for addresses table inside the address.py file
* Ran migration and upgraded the database



## Steps to test
1. Access the database - docker exec -it cas_py_backend /bin/bash
2. Run the command - SELECT * FROM addresses;
3. If you see the table addresses with the necessary columns, then the change should have been made.
